locales-src/asrouter.ftl

```diff
@@ -86,6 +86,10 @@ cfr-protections-panel-link-text = Learn more
 
 ## What's New toolbar button and panel
 
+# This string is used by screen readers to offer a text based alternative for
+# the notification icon
+cfr-badge-reader-label-newfeature = New feature:
```

Comment:
I'm confused: if it's read by screen readers, is the colon relevant, or actually detrimental?